Tests/conf.json

```diff
@@ -1324,7 +1354,7 @@
         },
         {
             "integrations": "ANYRUN",
-            "playbookID": "Detonate File From URL - ANYRUN - Test"
+            "playbookID": "Palo Alto Networks Cortex Test"
```

Comment:
why did you change this??

Reply:
Mistake, I was having lots of conflicts in my PR and must have thought this was accident that I made.